src/app/button-test/page.tsx

```diff
@@ -0,0 +1,8 @@
+'use client';
+
+import React from 'react';
+import BigButton from '@/components/BigButton';
+
+export default function ButtonTest() {
+  return <BigButton>Big Button</BigButton>;
+}
```

Comment:
Nice! I think we should remove testing pages before merging them in ideally though, but maybe you can comment this out and move it into `BigButton.tsx` to show how to use it?

Reply:
+1! let's keep this here for right now because we're not ready to merge yet. once we are, we can do what Jinkang recommended

Reply:
Also, let's pass in some props to the button so we can test it:

```
<BigButton color={COLORS.activeGreen} onChange={() => console.log('button clicked')}>
      Big Button
</BigButton>
```